packages/template-retail-react-app/app/hooks/use-auth-modal.js

```diff
@@ -342,7 +342,7 @@ AuthModal.propTypes = {
  */
 export const useAuthModal = (initialView = LOGIN_VIEW) => {
     const {isOpen, onOpen, onClose} = useDisclosure()
-    const {passwordless, social} = getConfig().app.login
+    const {passwordless = {}, social = {}} = getConfig().app.login || {}
```

Comment:
In the code below you are returning an object that has `isPasswordlessEnabled: passwordless?.enabled,` if you default in an empty object, then enabled will be undefined, which is a falsey values.. but you might want to return something like `isPasswordlessEnabled: !!passwordless?.enabled,`  instead. This goes for the other `isSocilEnabled` value too.